README.md

```diff
@@ -612,6 +612,11 @@ eval_sample_packing:
 sample_packing_eff_est:
 total_num_tokens:
 
+# Passed through to transformers when loading the model when launched without accelerate
+device_map:
```

Comment:
Would it be possible that you add the mention of using `sequential` here if you want to limit memory? Also, since this seems to be only for single gpu. A note here would be good.

Reply:
Thanks @winglian 

@NanoCode012 This is indeed for use across multiple gpus and is how I use it to control memory per device.